src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+
+    public double getArea();
```

Comment:
```suggestion
    double getArea();
```

все абстрактные методы в интерфейсах по умолчинаю публичные